Modules/Core/ImageAdaptors/test/itkVectorImageTest.cxx

```diff
@@ -293,10 +293,10 @@ itkVectorImageTest(int, char * argv[])
       itk::TimeProbe clock;
       clock.Start();
 
-      auto                                     image = VariableLengthVectorImageType::New();
+      auto image = VariableLengthVectorImageType::New();
       VariableLengthVectorImageType::IndexType start;
-      InternalPixelType                        f(VectorLength);
-      VariableLengthVectorImageType::SizeType  size;
+      InternalPixelType f(VectorLength);
+      VariableLengthVectorImageType::SizeType size;
```

Comment:
@thewtex Thanks for your feedback! It's obviously a matter of taste, but I find this proposed change an improvement of the readability. Old code (having enabled declaration alignment):

```cpp
      auto                                     image = VariableLengthVectorImageType::New();
      VariableLengthVectorImageType::IndexType start;
      InternalPixelType                        f(VectorLength);
```

Proposed with this pull request:
```cpp
      auto image = VariableLengthVectorImageType::New();
      VariableLengthVectorImageType::IndexType start;
      InternalPixelType f(VectorLength);
```

Reply:
Yes, when there are fewer items, it is more of a preference. This example, with fewer items and highly varying type lengths, is more compact. In general, it is easier to read when they are aligned.

More significantly, when there are many items, it is much easier to identify the types and values visually aligned.